src/main/webapp/scholarx/past-scholarx.html

```diff
@@ -0,0 +1,200 @@
+    <!-- Favicon -->
+    <link href="/assets/img/brand/favicon.png"
+          rel="icon"
+          type="image/png">
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700"
+          rel="stylesheet">
+    <!-- Icons -->
+    <link href="/assets/vendor/nucleo/css/nucleo.css"
+          rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
+          rel="stylesheet">
+    <!-- Argon CSS -->
+    <link type="text/css"
+          href="/assets/css/argon.css?v=1.0.1"
+          rel="stylesheet">
+    <!-- global CSS -->
+    <link type="text/css"
+          href="/assets/css/global-custom.css"
+          rel="stylesheet">
+
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap"
+          rel="stylesheet">
+</head>
+
+<body style="font-family: 'Montserrat', sans-serif; color: black">
+<header class="header-global">
+    <!--nav bar goes here-->
+    <div id="navbar"></div>
+</header>
+
+<section class="section pb-0 bg-gradient-white">
+    <div class="container">
+        <div class="row row-grid align-items-center justify-content-center">
+            <div class="col-md-6">
+                <div class="d-flex px-3">
+                    <div class="pl-4">
+                        <img src="/assets/img/brand/scholarx.png"
+                             class="img-fluid align-self-center mb-5 w-50">
+                        <p class="pb-5">ScholarX is an exclusive 3-month program aimed at providing mentoring support to
+                            a selected pool of High Potential undergraduate students based in Sri Lanka ideally by a Sri
+                            Lankan expat currently engaged with one of the world’s top universities or Fortune 500
+                            companies. It’s our free premium mentoring platform by Sri Lankans for Sri Lankans working
+                            towards creating a culture of knowledge and expertise sharing without the limitation of
+                            geographical borders.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-6">
+                <img src="images/hero-image.svg"
+                     class="img-fluid align-self-center w-75">
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="section bg-secondary-light" id="past-scholarx">
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-9 text-center">
+                <h1 class="mb-0 text-onelive">Take a look at our past</h1>
+                <p class="display-3 text-onelive font-weight-bolder">ScholarX</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-6">
+                <img class="card-img"
+                     src="images/card-image.jpg">
+                <div class="card" style="width: auto;">
```

Comment:
Try to use bootstrap as much as you can and use CSS only by adding classes to the global-custom CSS file. Avoid using inline CSS styles.

Reply:
@Gravewalker666  Now you have to edit styles.css file in the same directory. If that rule is used multiple files. you have to move them for the immediate parents style.css file.